versions/3.0.md

```diff
@@ -183,6 +181,18 @@ license:
 version: 1.0.1
 ```
 
+#### <a name="hostObject"></a>Host Object
```

Comment:
I figure there is a big documentation update once we agree on the merges, will make sure this is taken into account.